### PR TITLE
feat(aapd-1502): add api routes for appellant submissions

### PR DESCRIPTION
**Pull request by Si3rr4wow**
## Ticket Number

https://pins-ds.atlassian.net/browse/AAPD-

## Checklist

- [ ] Requires infrastructure changes
- [ ] I have updated the documentation accordingly
- [ ] My commit history in this PR is linear
- [ ] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `main` (please only [rebase](https://github.com/Planning-Inspectorate/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

**Review by bryn-pins — CHANGES_REQUESTED** [review]
looks good, can we get some tests in though please

**Comment by Si3rr4wow** [comment]
@bryn-pins ah yep sorry, I forget about them when they're sequestered away in their own dir

File: packages/appeals-service-api/src/routes/v2/appellant-submissions/appeallant-submissions.spec.js
Comment (DeanH1990) [inline packages/appeals-service-api/src/routes/v2/appellant-submissions/appeallant-submissions.spec.js:1]: small typo in the file name

**Review by DeanH1990 — DISMISSED** [review]
Looks good to me!

**Review by DeanH1990 — APPROVED** [review]

**Pull request merged**